### PR TITLE
Bugfix/contacts

### PR DESCRIPTION
**Pull request by h9k**

### CODE REVIEW

**Review by papakai — CHANGES_REQUESTED** [review]
Use double quotes over single quotes for array key access.

**Comment by h9k** [comment]
> Use double quotes over single quotes for array key access.

done!

**Review by papakai — APPROVED** [review]

File: modules/registrars/keysystems/lib/RRPProxyClient.php [hunk 1]
```diff
@@ -92,7 +92,20 @@ public function getContactInfo($contact)
                 "Email" => $response["property"]["email"][0]
             ];
         } catch (\Exception $ex) {
-            return [];
+            return [
+                "First Name" => '',
```
Comment (papakai) [inline modules/registrars/keysystems/lib/RRPProxyClient.php:96]: Keep empty Strings please also over double quotes. thanks!

**Review by papakai — CHANGES_REQUESTED** [review]
See my inline comment. ty

File: modules/registrars/keysystems/lib/RRPProxyClient.php (hunk 1 above)
Comment (h9k) [reply modules/registrars/keysystems/lib/RRPProxyClient.php]: ok, done!

**Pull request merged**